packages/react-core/src/components/WizardComposable/examples/WizardComposable.md

```diff
@@ -0,0 +1,58 @@
+---
+id: Wizard
+section: components
+cssPrefix: pf-c-wizard
+propComponents:
+  [
+    'WizardComposable',
+    'WizardComposableFooter',
+    'WizardComposableToggle',
+    'WizardComposableStep',
+    'WizardComposableBody',
+  ]
+---
+
+import { Button, WizardComposable, WizardComposableFooter, WizardComposableStep, WizardComposableBody, useWizardFooter, ModalVariant, Alert, EmptyState, EmptyStateIcon, EmptyStateBody, EmptyStateSecondaryActions, Title, Progress } from '@patternfly/react-core';
+import { css } from '@patternfly/react-styles';
+import styles from '@patternfly/react-styles/css/components/Wizard/wizard';
+import ExternalLinkAltIcon from '@patternfly/react-icons/dist/esm/icons/external-link-alt-icon';
+import SlackHashIcon from '@patternfly/react-icons/dist/esm/icons/slack-hash-icon';
+
+PatternFly has two implementations of a `Wizard`, where the latest `WizardComposable`. Its temporary name speaks to the improvements made to the composition of the Wizard and its ansillary components.
```

Comment:
```suggestion
PatternFly has two implementations of a `Wizard`.

 The first is the newest `WizardComposable`. It takes a more explicit and declarative approach. Its temporary name speaks to the improvements made to the composition of the Wizard and its ancillary components.
```

Reply:
That first sentence I had definitely didn't sound right, so thanks for this suggestion.